gui/src/pages/Sandbox.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import Layout from '../components/Layout.svelte';
+  import IfEnabled from '../components/IfEnabled.svelte';
+</script>
+
+<Layout>
+  <h1>Sandbox</h1>
+
+  <div>
+    <h2>Feature Flags</h2>
+    <IfEnabled feature="sandbox test">
```

Comment:
@jwmza This is just an example of how to use the `IfEnabled` component

Reply:
LGTM, thanks :)